src/containerapp/azext_containerapp/tests/latest/test_containerapp_env_commands.py

```diff
@@ -460,6 +460,34 @@ def test_containerapp_env_internal_only_e2e(self, resource_group):
         ])
 
 
+    @ResourceGroupPreparer(location="eastus")
+    @live_only()  # passes live but hits CannotOverwriteExistingCassetteException when run from recording
+    def test_containerapp_env_infrastructure_rg(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
```

Comment:
please check if this config would impact other modules.

Reply:
Hi @yanzhudd 
Currently will not impact other modules if run serially.
Will add this fix as core:
[{containerapp} add random config directory for test by Greedygre · Pull Request #27382 · Azure/azure-cli (github.com)](https://github.com/Azure/azure-cli/pull/27382)